src/sandbox/jakub/login/index.tsx

```diff
@@ -1,19 +1,20 @@
-import { yupResolver } from '@hookform/resolvers/yup/dist/yup'
-import { Button, TextField, TextFieldProps } from '@mui/material'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { Button } from '@mui/material'
 import React from 'react'
-import { Controller, useForm } from 'react-hook-form'
+import { useForm } from 'react-hook-form'
 
 import Form from 'components/form'
+import TextInput from 'components/textInput'
 
 import { LoginForm, LoginFormSchema } from './schema'
 
 const LoginPage: React.FC = () => {
   const methods = useForm<LoginForm>({
     reValidateMode: 'onSubmit',
```

Comment:
Jeżeli miałeś na myśli ten error w konsoli: `A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component.`

Dzieje się tak dlatego, że domyślnie wszystkie wartości są `undefined` i jak zaczynamy coś wpisywać to zmieniają się na `defined -> string` dlatego jest ten warning.
Żeby temu zapobiec możemy dodać `defaultValues` do `useForm()`, gdzie `defaultValues` musi być obiektem o typie `LoginForm`
Możesz stworzyć plik `/login/utils.ts` w którym będzie funkcja `getLoginFormDefaultValues()`, która będzie zwracała obiekt o typie `LoginForm`, gdzie domyślnymi wartościami będą puste stringi : `""`

```suggestion
  const methods = useForm<LoginForm>({
    defaultValues: getLoginFormDefaultValues(),
    reValidateMode: 'onSubmit',
    ...
```